frontend/src/scenes/insights/InsightContainer.tsx

```diff
@@ -202,6 +202,18 @@ export function InsightContainer({
                 className="insights-graph-container"
             >
                 <div>
+                    {isFunnelsFilter(filters) ? (
+                        <Row
```

Comment:
ideally we'd avoid adding more antd and use `<div className="flex flex-col"` or `<div className="flex flex-row` as appropriate but IIRC there're loads of `Row` and `Col` in insights so two more is 🤷

Reply:
updated, thanks